step-release-vis/src/app/services/environment_test.ts

```diff
@@ -68,6 +68,88 @@ describe('EnvironmentService', () => {
     expect(result).toEqual(line);
```

Comment:
can we have a test for a case where all polygon should be closed pls?

Reply:
but doesn't this mean that all of the candidates have 0 job count? we close all the polygons in the calculatePolygons function(not added yet) if they still have more than 0 active jobs. In closePolygons we close just the ones that get to 0 jobs, it's treated as a special case in calculatePolygons for the remaining candidates at the end with active jobs.
+ @laura-georgescu